osbs/build/user_params.py

```diff
@@ -125,6 +126,7 @@ def set_params(self,
                 self.build_imagestream.value = source_value
 
         self.parent_images_digests.value = parent_images_digests
+        self.allowed_platform.value = allowed_platform
```

Comment:
you'll also need to add allowed_platform to atomic-reactor's user_params schema.

Reply:
Done in https://github.com/projectatomic/atomic-reactor/pull/1177
Thanks for catching this one :)